JSDemos/Demos/Toolbar/Multiline/jQuery/index.js

```diff
@@ -0,0 +1,390 @@
+const toolbarDataDefault = [
```

Comment:
Why `Default`? Data is the most common name u can assign to a variable
```suggestion
const toolbarItems = [
```

Reply:
https://github.com/DevExpress/devextreme-demos/pull/2004/commits/05c2a4c6f2f88d68152ae92b286c4eba0e1284b7